README.md

```diff
@@ -55,5 +55,6 @@ dotnet test
 
 ---
 
-For the OMF landing page [ReadMe](../../../)  
+For the main OCS bulk uploads samples page on master [ReadMe](https://github.com/osisoft/OSI-Samples-OCS/blob/master/docs/BULK_UPLOAD_README.md)  
+For the main OCS samples page on master [ReadMe](https://github.com/osisoft/OSI-Samples-OCS)
 For the OSIsoft Samples landing page [ReadMe](https://github.com/osisoft/OSI-Samples)
```

Comment:
this needs a carriage return. the two linkbacks are on the same line

Reply:
I think this is normally done here using two spaces at the end of the previous line to force the carriage return.